services/circleci/circleci.tester.js

```diff
@@ -25,7 +25,7 @@ t.create('circle ci (valid, with branch)')
 
 t.create('circle ci (not found)')
   .get('/project/github/PyvesB/EmptyRepo.json')
-  .expectJSON({name: 'build', value: 'Project not found'});
+  .expectJSON({name: 'build', value: 'not found'});
```

Comment:
I like the explicit message. You can pass a custom "not found" message to `checkErrorResponse` to preserve the old behavior.

Reply:
Reverted. I've nevertheless switched to a lower case `P` to be more consistent with what we use elsewhere.